packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css

```diff
@@ -83,9 +83,6 @@
   height: 25px;
   border-radius: 50%;
 }
-.world:hover {

```

Comment:
You've just done away with the code. So how will a user know if they've hovered on the icon?
I suggest you add a better hover effect, not just remove the code entirely.

Reply:
Ok @JulianaSau  What of if i add a scale effect, So the icon grows bigger in a nice way when hovered with a good transition?

Reply:
Yes. That would work.
You could also make the opacity of the color of the icon change when it's hovered on

Reply:
You'll tag me when you're done to review your changes again

Reply:
Ok Making changes now

Reply:
@JulianaSau  I'm done should i create a new pull request for it?

Reply:
Hi @Favourz1, Send a video of how it behaves now

Reply:
@JulianaSau  Please find attached link to the video and how the icon works now.

https://drive.google.com/file/d/1AdGbkH38nzS6WlG1yTgm0V6w-TdMXwml/view?usp=drivesdk